songbird/integration/hikari.py

```diff
@@ -1,41 +1,59 @@
-# mypy: ignore-errors
-
 from __future__ import annotations
 
-from typing import Callable, Awaitable, Any
+from typing import (
+    Callable,
+    Awaitable,
+    Any,
+    TypeVar,
+    Type,
+)
 
 from hikari import snowflakes, VoiceEvent, GatewayBot
 from hikari.api import VoiceComponent, VoiceConnection
 
-from .songbird import Driver
-from .voicebox_base import VoiceboxBase
+from ..songbird import Driver
+from songbird.integration.voicebox_base import VoiceboxBase
+
+
+VoiceBoxType = TypeVar("VoiceBoxType", bound="HikariVoicebox")
 
 
-class Voicebox(VoiceboxBase, VoiceConnection):
-    """Hikari VoiceConnection using Songbird"""
+class HikariVoicebox(VoiceboxBase, VoiceConnection):
```

Comment:
This name shouldn't be changed. Its a breaking change.

Reply:
Maybe, but it is more convenient to use it with this name in the project